Updated support data: <picture>

DCoder18 opened the pull request:
The expected result is we see the appropriate placeholder images (large to small) according to the screen size. If this is not supported, we should see a placeholder image with the **Default** text on it.

In Apple Mail, there is an odd issue where the `picture` tag and its child elements gets stripped along with any sibling elements in certain use cases. I have included testi links below with some of the use cases.

- [picture tag with img](https://testi.at/proj/9zj3i524t5xz1be2u9)
 `picture` and `img` tags are stripped.

- [picture tag with text wrapped in div](https://testi.at/proj/ov530ajr0gb2fv341lk)
   This works as expected. However, an empty `div` tag strips the `picture` tag again.

I tried using `img` and `<div>Text</div>` together with `picture`. In this case, only the `img` tag is stripped. The `picture` and `div` tags remain and works as expected.

hteumeuleu commented:
I'm very curious about this one! Could you elaborate what you mean by ”_`<picture>` tag is stripped in some cases._”? Even when glancing at the results of your Testi.at tests, I can't see what's wrong in Apple Mail.

DCoder18 commented:
I've just updated the PR description now. As you can see, the results are quite strange and inconsistent. I cannot pin point what is exactly causing this issue yet. I do agree, the note can be made more clear. Do you have any suggestions on how I can improve the note I added?

hteumeuleu commented:
@DCoder18 Could you share the code from the first faulty test?

DCoder18 commented:
Here you go. In this case, both the `picture` and `img` tags are stripped in Apple Mail.

```html
<!DOCTYPE html>
<html>
	<head>
		<title>picture element</title>
	</head>
	<body>
		<h1>picture</h1>
		
		<h1><code>picture</code> with <code>img</code></h1>
		<picture>
			<source srcset="https://40daystemp3.testi.at/placeholder/600x600/F4F4F4/5a5a5a.png?text=Small" media="(max-width: 480px)" />
			<source srcset="https://40daystemp3.testi.at/placeholder/600x600/F4F4F4/5a5a5a.png?text=Medium" media="(max-width: 640px)" />
			<source srcset="https://40daystemp3.testi.at/placeholder/600x600/F4F4F4/5a5a5a.png?text=Large" media="(min-width: 641px)" />
			<img src="https://40daystemp3.testi.at/placeholder/600x600/F4F4F4/5a5a5a.png?text=Default" alt="" style="display:block; width:100%; height:auto;" />
		</picture>
		

		<img src="https://40daystemp3.testi.at/placeholder/600x600/F4F4F4/5a5a5a.png?text=Default" alt="" style="display:block; width:100%; height:auto;" />
	</body>
</html>
```

hteumeuleu commented:
@DCoder18 Thank you! It looks like to make this email work properly, all that's needed is to add background to the body (`<body style="background:#fff;">`). I think this is one of those cases where the email client has some heuristics to determine whether it's an HTML newsletter or a text email sent as HTML. 

There might also be a problem with your image host (`https://40daystemp3.testi.at/`). It looks like it doesn't allow CORS requests (see https://parcel.io/tools/cors-check). Maybe this is why it does not work. Overall, I think this is a specific problem to the specific combination of "simple email + cors issue" and not worth a downgrade of support. (But perhaps a note indicating that images blocked by CORS are not downloaded?)
